### PR TITLE
Add About Buildout

### PR DESCRIPTION
**Pull request by navdeep300**
I have added some points about buildout, and also fixed a minor typo

### CODE REVIEW

File: docs/conf.py~ [hunk 1]
```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
```
Comment (sigmavirus24) [inline docs/conf.py~]: Your emacs is configured poorly and you added the swap file. Please remove this.

File: docs/scenarios/admin.rst~ [hunk 2]
```diff
@@ -0,0 +1,351 @@
+Systems Administration
```
Comment (sigmavirus24) [inline docs/scenarios/admin.rst~]: Again, you need to remove this.

**Comment by sigmavirus24** [comment]
Please remove the `~` files from emacs and let me know when you have done that so I can review it again.

**Comment by navdeep300** [comment]
I have done the changes please review and suggest any improvements.

**Comment by sigmavirus24** [comment]
Thanks @navdeep300!

**Pull request closed without merging**